about.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html> <!--lang attr here?-->
+    <head>
+        <meta charset="UTF-8" />
+        <title>Lauren Covington</title>
+        <link rel="stylesheet" href="stylesheets/style.css">
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Shadows+Into+Light&display=swap" rel="stylesheet">
+    </head>
+
+    <body>
+        <nav id="navbar">
+            <a href="#about">About</a>
+            <a href="#portfolio">Portfolio</a>
+            <a href="#code-journal">Code Journal</a>
```

Comment:
Your links didn't go to your other pages because you used anchor links instead of relative URLs.

```suggestion
            <a href="./about.html">About</a>
            <a href="./portfolio.html">Portfolio</a>
            <a href="./code-journal.html">Code Journal</a>
```